Use fixed port number for our python tests

FrostyX opened the pull request:
Randomly generated port numbers worked just find on developer's
machines but Koji have issues with some of the numbers. Let's not
burn hours trying to figure out which ports work and which doesn't
and just settle on one fixed number. It is more deterministic and
reproducible anyway.

clime commented:
Thank you! Could we maybe do 8092 instead of 8080? Because 8080 could somebody use for local web development.

FrostyX commented:
Unfortunately `8092` didn't work. I changed it to `8888`. We might experiment a bit more if you want to.

I've also reverted the commit disabling tests in spec file.

clime commented:
Cool, thanks.
